src/components/Card.js

```diff
@@ -72,39 +67,43 @@ class Card {
   }
```

Comment:
```js


    this._cardElement // handle a click on the trashcan
      .querySelector(".card__button-del")
      .addEventListener("click", () => {
        this._handleDeleteClick(this, this._id);
      });
```

You've defined `this._trashEl` in line 33. Please, use it here


```js

      this._trashEl.addEventListener("click", () => {

```

Reply:
Corrected